index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello-Microverse-Project</title>
+</head>
+<body>
+  <h2 class="header">
```

Comment:
Kindly, use `h1` tag instead of  `h2` tag as required in the project requirements.
![curriculum-transversal-skills_hello_microverse_project md at main · microverseinc_curriculum-transve](https://user-images.githubusercontent.com/71507900/147237921-7474878a-fded-4b48-a768-2b0cfffc6a35.png)